src/go/cmd/api_view_test.go

```diff
@@ -187,11 +183,13 @@ func TestAliasDefinitions(t *testing.T) {
 			diagLevel:  DiagnosticLevelWarning,
 			name:       "external_package",
 			path:       "testdata/test_external_alias_exporter",
-			sourceName: "github.com/Azure/azure-sdk-for-go/sdk/test_external_alias_source.Foo",
+			sourceName: "github.com/Azure/azure-sdk-tools/src/go/cmd/testdata/test_external_alias_source.Foo",
 		},
 	} {
 		t.Run(test.name, func(t *testing.T) {
-			review, err := createReview(filepath.Clean(test.path))
+			p, err := filepath.Abs(test.path)
+			require.NoError(t, err)
```

Comment:
`Abs` calls `Clean`. We need an absolute path in this test because I removed the hook we formerly used to manipulate the code that finds local modules. That code now works in general cases--it doesn't assume it's always looking for SDK modules--so it doesn't require fakery and using an absolute path here tests that.